backend/Squirrel.SqlService/Squirrel.SqlService.WebApi/Controllers/ConsoleAppHubController.cs

```diff
@@ -171,4 +171,13 @@ await _hubContext.Clients.User(queryParameters.ClientId)
             .GetUserDefinedTableTypesAsync(_queryParameters.queryId);
         return Ok(await _queryParameters.tcs.Task);
     }
+    
+    // https://localhost:7244/api/ConsoleAppHub/getUserDefinedTableTypes
+    [HttpPost("dbConnect")]
```

Comment:
if the comment shows the full route to the endpoint, it is wrong)
also it's better to use dashes (db-connect) than camelCase in the routes. And not to use verbs like "get" in the previous routes